documentation/specifications/transactions/onchain/README.md

```diff
@@ -1,11 +0,0 @@
-# On-Chain Data and Transactions
-
-Aptos transactions mutate and create state (or resources) within the set of [on-chain modules](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/framework/core/sources), primarily the [Aptos Account](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/doc/AptosAccount.md). The transaction format is defined in the [Move Adapter Specification](https://github.com/aptos-labs/aptos-core/blob/main/specifications/move_adapter/README.md). Most participants of the Aptos Blockchain will submit SignedTransactions containing a [script function](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/script_documentation/script_documentation.md). Before release 1.2, clients used scripts. These can be accessed in [compiled form](https://github.com/aptos-labs/aptos-core/tree/release-1.1/language/stdlib/compiled/transaction_scripts) and in their [original form](https://github.com/aptos-labs/aptos-core/tree/release-1.1/language/stdlib/transaction_scripts). The Aptos Blockchain only allows script functions and this set of pre-registerd scripts to be submitted. Due to the evolving nature of Move and the Move compiler, compiling existing scripts may not result in the form stored in the directory stored above. Hence, it is recommended to use script functions where available or otherwise the compiled scripts.
```

Comment:
This appears to be the only thing of value in the `aptos-core/documentation/specifications/transactions/*` directory and even then, it's out of date and incorrect. I think our new developer documents cover this in a better way.